ClearKeep/Scenes/Register/Views/RegisterContentView.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  RegisterContentView.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 04/03/2022.
+//
+
+import SwiftUI
+import CommonUI
+import RealmSwift
+
+private enum Constants {
+	static let width = 120.0
+	static let height = 40.0
+	static let radius = 20.0
+	static let sapcing = 20.0
+}
+
+struct RegisterContentView: View {
+	// MARK: - Variables
+	@Environment(\.colorScheme) var colorScheme
+	@Binding var username: String
+	@Binding var password: String
+	@Binding var displayname: String
+	@Binding var rePassword: String
+	@Binding var inputStyle: TextInputStyle
+	// MARK: - Body
+	var body: some View {
+		GroupBox(label:
+					Text("Register.Please fill in the information below to complete your sign up".localized)
```

Comment:
@Minhdn1 : Localize string key does not contain space
```suggestion
					Text("Register.Title".localized)
```

Reply:
fix complete